CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ Versions are `MAJOR.PATCH`.
   unbootstrap chocolatey. [#64722](https://github.com/saltstack/salt/issues/64722)
 - Add Ubuntu 24.04 support [#66180](https://github.com/saltstack/salt/issues/66180)
 - Add Fedora 40 support, replacing Fedora 39 [#66300](https://github.com/saltstack/salt/issues/66300)
+- Added ssl options to postgres pillar [#67720](https://github.com/saltstack/salt/issues/67720)
```

Comment:
Please see the documentation [here](https://github.com/saltstack/salt/blob/master/CONTRIBUTING.rst#changelog-and-commit) for creating changelog entries. We don't modify the changelog file directly. Instead, create a file in the changelog directory with the issues you're fixing. Ie: `67720.added.md` and put the description in that file. When we do a release, that will be added to the changelog using [towncrier](https://github.com/twisted/towncrier).